notifications_utils/pdf.py

```diff
@@ -1,17 +1,16 @@
-import PyPDF2
-from PyPDF2 import PdfWriter
-from PyPDF2.errors import PdfReadError
+from pypdf import PdfWriter
+from pypdf.errors import PdfReadError
 import io
 
 
 def pdf_page_count(src_pdf):
     """
     Returns number of pages in a pdf file
 
-    :param PyPDF2.PdfReader src_pdf: A File object or an object that supports the standard read and seek methods
+    :param pypdf.PdfReader src_pdf: A File object or an object that supports the standard read and seek methods
     """
     try:
-        pdf = PyPDF2.PdfReader(src_pdf)
+        pdf = pypdf.PdfReader(src_pdf)
```

Comment:
Where is `pypdf` being imported? Shouldn't you just add this import in the first line? See my comment above.